src/core/thread/mle_types.hpp

```diff
@@ -161,6 +161,8 @@ constexpr uint8_t kRouterSelectionJitter      = 120; ///< (in sec)
 constexpr uint8_t kRouterDowngradeThreshold = 23;
 constexpr uint8_t kRouterUpgradeThreshold   = 16;
 
+constexpr uint8_t kChildRouterLinks = OPENTHREAD_CONFIG_MLE_CHILD_ROUTER_LINKS;
```

Comment:
I'm not sure this variable is useful, why not use `OPENTHREAD_CONFIG_MLE_CHILD_ROUTER_LINKS` directly.

Reply:
Thanks for the explaination.